scalastyle-config.xml

```diff
@@ -95,7 +95,7 @@
  </check>
  <check level="error" class="org.scalastyle.scalariform.NumberOfTypesChecker" enabled="true">
   <parameters>
-   <parameter name="maxTypes"><![CDATA[30]]></parameter>
+   <parameter name="maxTypes"><![CDATA[40]]></parameter>
```

Comment:
It's probably better to add a scalastyle exception for one file than to globally increase this limit.

Reply:
Added scalastyle:off and reverted the limit